updated adding instance_format to variable query (#83)
nasa/python_cmr: CHANGELOG.md

- Support multi-point searches ([#72](https://github.com/nasa/python_cmr/issues/72))
- Support `processing_level_id` in `CollectionQuery` ([#76](https://github.com/nasa/python_cmr/issues/76))
- Support `platform` in `CollectionQuery` ([#77](https://github.com/nasa/python_cmr/issues/77))

> - Support `instance-format` to `VariableQuery` to allow for searching of variable zarr stores ([#59]https://github.com/nasa/python_cmr/issues/59)

Reviewer: ```suggestion
- Support searching by instance format for `VariableQuery` ([#59]https://github.com/nasa/python_cmr/issues/59)
```
